forest/src/daemon.rs

```diff
@@ -31,14 +31,12 @@ use std::time;
 /// Starts daemon process
 pub(super) async fn start(config: Config) {
     // Set the Address network prefix
-    #[cfg(any(feature = "testnet"))]
-    address::NETWORK_DEFAULT
-        .set(address::Network::Testnet)
-        .unwrap();
-    #[cfg(not(feature = "testnet"))]
-    address::NETWORK_DEFAULT
-        .set(address::Network::Mainnet)
-        .unwrap();
+    let network = if config.chain.name == "mainnet" {
```

Comment:
It would be nice to have `From<String>` (or similar for str) implemented for `address::Network`. Then we would just need to `let network = "mainnet".into()` which would also check entry correctness instead of relying on `clap`. The less logic in the app itself, the easier it is to follow. :)